modules/moderation.py

```diff
@@ -610,7 +610,7 @@ async def _strike_set(self, ctx, member: discord.Member, count: StrikeRange, *,
                 return await ctx.message.delete()
 
             await ctx.send(
-                f'{config.greenTick} {activeStrikes - count} strikes for {member} ({member.id}) have been successfully removed. they now have {count}.'
+                f'{config.greenTick} {activeStrikes - count} strikes for {member} ({member.id}) have been successfully removed, they now have {count} strikes'
```

Comment:
The "math" should be added to this command too: "they now have X strikes (X - X)"